src/multibanco/multibanco.js

```diff
@@ -0,0 +1,30 @@
+// 0. Get originKey
+getOriginKey().then(originKey => {
+    getPaymentMethods().then(paymentMethodsResponse => {
+        // 1. Create an instance of AdyenCheckout
+        const checkout = new AdyenCheckout({
+            originKey, // Mandatory. originKey from Costumer Area
+            environment: 'test',
+            amount: { currency: 'EUR', value: 1000 },
+            onAdditionalDetails: result => {
+                console.log(result);
+            },
+            onError: error => {
+                console.log(error);
+            },
+            onSubmit: initialSubmit,
+            paymentMethodsResponse
+        });
+
+        function initialSubmit (state, component) {
+            makePayment(state.data).then(response => {
+                initialComponent.unmount();
```

Comment:
```suggestion
                component.unmount();
```

The second parameter on the `onSubmit` event refers to the component that called it. In this case it would not be necessary to store it in a variable.